hw03_Vazquezsotoj.py

```diff
@@ -0,0 +1,155 @@
+#################################################################################
+# Author: Javier Vazquez Soto
+# Username:Vazquezsotoj
+#
+# Assignment:hw03
+# Purpose: To create something complex like a house,animal, or person
+# Google Doc Link: https://docs.google.com/document/d/1f5hiQ8grzoAFKAAftUvUoYdcgWC0BoQAqeDspi7zWOU/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#T03_Function_house.py
+#Hw02-loopy-turtles-loopy-languages.py
+#################################################################################
+
+import turtle
+
+def Ear1(x):
```

Comment:
Another gray squiggly. Functions names should be lower case (to avoid confusing them with class names, which are always upper case, like `Screen()` and `Turtle()`)